source/JsMaterialX/CMakeLists.txt

```diff
@@ -111,7 +111,7 @@ target_link_libraries(JsMaterialXGenShader
     PUBLIC MaterialXCore
     PUBLIC MaterialXFormat
     PUBLIC MaterialXGenShader
-    PUBLIC MaterialXGenEssl
+    PUBLIC MaterialXGenGlsl
```

Comment:
Don't understand this

Reply:
There is no new MaterialXGenEssl. Just reuse the existing library.